src/core/referencingfeaturelistmodel.h

```diff
@@ -33,6 +33,8 @@ class ReferencingFeatureListModel : public QAbstractItemModel
 
     Q_PROPERTY( QgsFeature feature WRITE setFeature READ feature NOTIFY featureChanged )
     Q_PROPERTY( QgsRelation relation WRITE setRelation READ relation NOTIFY relationChanged )
+    Q_PROPERTY( QString currentRelationId WRITE setRelationId READ relationId NOTIFY relationChanged )
```

Comment:
Is this a kind of convention? In my opinion it's more clear calling it `relationId` or `setCurrentRelationId` etc.

Reply:
I had to rename it, since "relationId" was already present in QML. I will rename all the cpp methods to have "current" in them.